src/components/Shared/Filters/FilterPanel.mocks.ts

```diff
@@ -148,7 +148,7 @@ export const savedFiltersMock = gqlMock<UserOptionFragment>(
+        '{"any_tags":false,"account_list_id":"08bb09d1-3b62-4690-9596-b625b8af4750","params":{"status":"never_contacted,ACTIVE,HIDDEN,NULL,partner_financial,Appointment Scheduled", "completed":"true", "overdue": "true", "primaryAddress": "primary, null", "reverseActivityType": "true", "reverseContactAppeal": "true", "reverseContactChurch": "true", "reverseContactCity": "true", "reverseContactCountry": "true", "reverseContactDesignationAccountId": "true", "reverseContactIds": "true", "reverseContactLikely": "true", "reverseContactMetroArea": "true", "reverseContactPledgeFrequency": "true", "reverseContactReferrer": "true", "reverseContactRegion": "true", "reverseContactState": "true", "reverseContactStatus": "true", "reverseContactTimezone": "true", "reverseContactType": "true", "reverseNextAction": "true", "reverseResult": "true", "reverseTags": "true", "reverseUserIds": "true", "contactChurch": "test1, test2", "contactCity": "test1", "contactCountry": "test1, test2", "contactDesignationAccountId": "test1, test2", "contactLikely": "test1, test2", "contactMetroArea": "test1, test2", "contactPledgeFrequency": "test1, test2", "contactReferrer": "test1, test2", "contactRegion": "test1, test2", "contactState": "test1", "contactTimezone": "test1, test2", "reverse_alma_mater": "false","contactNewsletter":"all","notes":{"wildcardNoteSearch": "note1"}, "activity_type": "appointment_in_person,Partner Care - Social Media", "result": "Attempted,Attempted - Left Message,Completed,Done,None,Received", "pledge_received":"true","pledge_amount":"35.0,40.0","pledge_currency":"USD","pledge_frequency":"0.46153846153846,1.0","pledge_late_by":"30_60","newsletter":"no_value","referrer":"d5b1dab5-e3ae-417d-8f49-2abdd915515b","city":"Evansville,Woodstock","state":"FL","country":"United States","metro_area":"Cool","region":"Orange County","contact_info_email":"Yes","contact_info_phone":"No","contact_info_mobile":"No","contact_info_work_phone":"No","contact_info_addr":"Yes","contact_info_facebook":"No","opt_out":"No","church":"Cool Church II","appeal":"851769ba-b55d-45f3-b784-c4eca7ae99fd,77491693-df83-46ec-b40b-39d07333f47e","timezone":"America/Vancouver","locale":"English","donation":"first","donation_date":"2021-12-23..2021-12-23","next_ask":"2021-11-30..2021-12-22","user_ids":"787f286e-fe38-4055-b9fc-0177a0f55947","reverse_appeal":true,"contact_types": "person"},"tags":null,"exclude_tags":null,"wildcard_search":""}',
```

Comment:
Should "Appointment Scheduled" be `appointment_scheduled`?

Reply:
Yes, now we should receive data from the API that is snakecase. I have that there to ensure it works just in case the data is formatted differently. Do you think we should remove it?

Reply:
Ok so yes "Appointment Scheduled" used to be a valid value. And I think it is possible that it could still be received in the future. So I will keep it.